Tools/gen_build_info.py

```diff
@@ -21,7 +21,7 @@
 
 VERSION = os.environ['GITHUB_SHA']
 FORK_ID = "nyanotrasen"
-BUILD_URL = f"https://builds.nyanotrasen.moe/{{FORK_ID}}/builds/{{FORK_VERSION}}/{FILE}"
+BUILD_URL = f"https://cdn.nyanotrasen.moe/builds/{{FORK_VERSION}}/{FILE}"
 MANIFEST_URL = f"https://cdn.nyanotrasen.moe/version/{{FORK_VERSION}}/manifest"
 MANIFEST_DOWNLOAD_URL = f"https://cdn.nyanotrasen.moe/version/{{FORK_VERSION}}/download"
```

Comment:
Is this something we want to change? What do they do?

Reply:
optional infrastructure we don't have

Reply:
We should probably tweak it just for the sake of not having their infrastructure in our repo. Comment it out, leave or blank, or whatever makes sense.